.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Bug Report
+<!--
+    If you are not submitting a bug report, ignore this template and see
+      http://Chapel-Issues-Best-Practices.com  (TODO)
+  -->
+### Summary of Problem
+
+
+### Steps to Reproduce
+
+**What behavior do you observe when encountering this issue?**
+
+**Please provide one of the following:**
+
+- Reduced program demonstrating issue *(preferred)*:
+- Original program demonstrating issue:
+- Description of how to recreate the issue *(last resort)*:
+
+**Please provide any of the following that are necessary to reproduce the issue:**
+
+- How to compile:
+- How to run:
+- Input files:
+- Helper modules:
+
+
+### General Information
+
+- `chpl --version`:
+- `$CHPL_HOME/util/printchplenv --anonymize`:
+- Back-end compiler version, e.g. `gcc --version` or `clang --version`:
+- *For Cray systems only*, output of `module list` :
```

Comment:
Note an inconsistency in the above:  The first two bullets simply give the command while the third and fourth essentially are descriptions.  I.e., I think bullets one and two should be "Output of ..." for consistency and maximal clarity.

For bullet 3, I might say "Back-end compiler and version" since the answer "5.4.2" wouldn't help if we didn't know which compiler it was.  In retrospect, I also think this is something we should build into the header of 'printchplenv' (would require us to teach each of our make/compiler/Makefile.* files identified the flag to use to determine the compiler's version).

For bullet 4, I'd say "(For Cray systems only)  Output of ..." in parens since it's more of a qualifier than part of what we're asking for.  Or I might say "On Cray systems, output of..."